app/Http/Controllers/Order/ExtendedBaseInvoiceController.php

```diff
@@ -137,7 +137,7 @@ public function multiplePayment($clientid, $invoiceChecked, $payment_method,
     {
         try {
             foreach ($invoiceChecked as $key => $value) {
-                if ($key != 0) {//If Payment is linked to Invoice
+                if (isset($key)) {//If Payment is linked to Invoice
```

Comment:
Show me this issue, through dd. Now with this change places where we are passing 0 as the key to skip this condition will start failing. Since isset will return true for even empty strings and 0